modules/client-configuration/pages/activation-keys.adoc

```diff
@@ -99,6 +99,19 @@ Click btn:[Generate New Key] to create the reactivation key.
 Record the details of the key for later use.
 Unlike typical activation keys, which are not associated with a specific system ID, keys created here do not show up on the menu:Systems[Activation Keys] page.
 
+=== Salt clients
+After creating a reactivation key, you can use it as the ``management_key`` grain in [path]``/etc/salt/minion.d/susemanager.conf``. For example:
+
+----
+grains:
+    susemanager:
+        management_key: "re-1-daf44db90c0853edbb5db03f2b37986e"
+----
+
+You can now restart the [command]``salt-minion`` to apply the reactivation key.
+
+=== Traditional clients
+
```

Comment:
I think the changes are fine, but I would prefer not to use the section headers for such a small section. Can we instead remove the headers, and just treat it as a para? Something like "For Salt clients, after you have created a reactivation key, ... " ?

Reply:
Sure thing @Loquacity ! I updated that in the latest commit.